_posts/2024-05-10-interactive-bioinformatics.md

```diff
@@ -0,0 +1,70 @@
+# Interactive bioinformatics
+
+This blog was suggested by Helena and inspired a question on gamification of bioinformatics. So buckle up as we look at a wide range of very cool educational tools and projects.
```

Comment:
judging by 'buckle up' this was llm authored? :laughing:

Reply:
:'( No LLM used.

Reply:
I learned something today, my apologies!